stories/topics/ref-aap-aws-install-prerequisites.adoc

```diff
@@ -27,8 +27,61 @@ You must have a working knowledge of the following aspects of {AWS}:
 
 For more information about {AWS} and terminology, see the link:https://aws.amazon.com/[AWS product documentation].
 
-Your AWS account must have _Identity and Access Management_ (IAM) permissions to create and manage {Market} deployments as well as the resources described in xref:con-aws-application-architecture[Application architecture]. 
-The application can fail to deploy if your IAM policies restrict deployment and management of these resources.
+Your AWS account must have _Identity and Access Management_ (IAM) permissions below to create and manage {Market} deployments as well as the resources described in xref:con-aws-application-architecture[Application architecture]. Your AWS account must also be licensed to deploy {PlatformNameShort} from AWS Marketplace.
+The application can fail to deploy if your IAM policies restrict deployment and management of these resources. The exact list of IAM policies necessary is found below.
+
+== IAM Policies
+
+*  Managed Policies
+** AWSCloudFormationReadOnlyAccess
+** AmazonS3ReadOnlyAccess
+** IAMReadOnlyAccess
+** AmazonRDSReadOnlyAccess
+** AmazonElasticFileSystemReadOnlyAccess
+** AmazonEC2FullAccess
+** AWSMarketplaceFullAccess
+* Cloudformation inline IAM policies
+** cloudformation:DeleteStack
```

Comment:
Is it Cloudformation or cloudformation?

and below

Reply:
The official IAM policy is lowercase, but CloudFormation is the product name